src/app/routes/getInitialData/index.js

```diff
@@ -1,5 +1,6 @@
 import 'isomorphic-fetch';
 import path from 'ramda/src/path';
+import Url from 'url-parse';
```

Comment:
this needs added to package.json (`npm i --save url-parse`)

Reply:
good spot

Reply:
Thanks - will install it now & update.